core/payment-driver/zksync/src/driver.rs

```diff
@@ -128,8 +128,9 @@ impl PaymentDriver for ZksyncDriver {
         _caller: String,
         msg: Exit,
     ) -> Result<String, GenericError> {
-        log::info!("EXIT = Not Implemented: {:?}", msg);
-        Ok("NOT_IMPLEMENTED".to_string())
+        wallet::exit(_caller, &msg)
```

Comment:
the `_` is in front of the param because its is in the "interface" but its not used yet.

When you start using it you can remove the `_`

Reply:
Oh, I think this is serious bug that we have overlooked. `_caller` **should be ignored**. This **not** the address to withdraw from. The address should be in the message (but it's not)!

Reply:
you are right, i was already wondering what `caller` was doing..

`msg.sender` should be used ( and checked if its in the `self. is_account_active(&msg.sender)` )